CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing
+
+The following is a set of guidelines for contributing to semeio.
+
+## Ground Rules
+
+1. We use Black code formatting
+1. We do our best to stick to pylint and flake8 code standards
+1. All code must be testable and unit tested.
+1. Commit messages should follow the format as described here https://chris.beams.io/posts/git-commit/
+
+## Pull Request Process
+
+1. Work on your own fork of the main repo
+1. Push your commits and make a draft pull request using the pull request template.
+1. Check that your pull request passes all tests.
```

Comment:
Remove . for uniform formatting. 

Add `tox -e py`
Check that your pull request passes all tests and is formatted appropriately.

Reply:
I did not add that as the procedure for running the tests is described in the README, and should we ever update that I dont think we should have to remember to do it several places.